ext/dom/tests/DOM4_DOMNode_after.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+DOMNode::after()
+--SKIPIF--
+<?php require_once('skipif.inc'); ?>
+--FILE--
+<?php
+require_once("dom_test.inc");
+
+$dom = new DOMDocument;
+$dom->loadXML('<test><mark>first</mark></test>');
```

Comment:
It would be good if there was another node after it, so you can see that the insertion position is actually correct.

Reply:
Added and realized its actually loosing nodes, need to fix the function.